utils/jpgicc/meson.build

```diff
@@ -0,0 +1,24 @@
+#project('jpegicc', 'c')
+
+jpegicc_srcs = [
+  'jpgicc.c',
+  'iccjpeg.c',
+  '../common/xgetopt.c',
+  '../common/vprf.c',
+  '../common/utils.h',
```

Comment:
You don't need to add headers, but it doesn't hurt. They mostly just affect generated IDE information.

The common/ source files, in general, should probably be compiled once and reused for all the utils (although this draft PR only implements jpgicc for now, you will probably want the rest eventually?).

What you could do is add `utils/common/meson.build` which:
- builds a [`static_library()`](https://mesonbuild.com/Reference-manual_functions.html#static_library),
- makes a [`declare_dependency()`](https://mesonbuild.com/Reference-manual_functions.html#declare_dependency) that links to the static library and adds the common directory as an `include_directories:`

and then link that dependency, let's call it `common_dep`, to each utility executable in the `dependencies:` kwarg.

Reply:
got it - that's a good way to do it. I'll put that together and squash our commits. (then I'll rinse and repeat for the other icc directories.)